src/JWTGuard.php

```diff
@@ -167,6 +170,11 @@ public function login(JWTSubject $user)
         $token = $this->jwt->fromUser($user);
         $this->setToken($token)->setUser($user);
 
+        // If we have an event dispatcher instance set we will fire an event so that
```

Comment:
> If we have an event dispatcher instance set

I don't read anything in the code where the event dispatcher is option; it's injected via DI and is required, so I think the comment isn't technically correct.

My other feedback here is: the method is called `fireLoginEvent`

The comment spawns 3 lines essentially telling us this.

My TL;DR feedback here would: method name is good and very clear, we don't need the comment at all

Reply:
Well i did just copy this from the SessionGuard without thinking about it. I removed it ^^